pages/chaoss/visualizations/contrib_importance_over_time.py

```diff
@@ -0,0 +1,416 @@
+                            ), 
+                        
+                                dbc.Col(
+                                    dbc.Input(
+                                            id=f"window-width-{PAGE}-{VIZ_ID}",
+                                            type="number",
+                                            min=1,
+                                            max=12,
+                                            step=1,
+                                            value=6,
+                                            size="sm",     
+                                        ),
+                                    className="me-2", 
+                                    width = 1,    
+                                ), 
+
+                                dbc.Label(
+                                "Step Size:",
+                                html_for=f"step-size-{PAGE}-{VIZ_ID}",
+                                width="auto",
+                            ), 
+                                dbc.Col(
+                                    dbc.Input(
+                                            id=f"step-size-{PAGE}-{VIZ_ID}",
+                                            type="number",
+                                            min=1,
+                                            max=12,
+                                            step=1,
+                                            value=6,
+                                            size="sm",     
+                                        ), 
+                                    className="me-2",
+                                    width=1,   
+                                ),
+                                dbc.Alert(
+                                    children="Please ensure that 'Step Size' is less than or equal to 'Window Size'",
+                                    id=f"check-alert-{PAGE}-{VIZ_ID}",
+                                    dismissable=True,
+                                    fade=False,
+                                    is_open=False,
+                                    color="warning",
+                                ),
+                            ], 
+                            align="center"
+                        ), 
+                        dbc.Row(
+                            [   dbc.Label("Filter Out Contributors with Keyword(s) in Login:",
+                                    html_for=f"patterns-{PAGE}-{VIZ_ID}",
+                                    width="auto",), 
+                                dbc.Col(
+                                    [
+                                        dcc.Dropdown(
+                                            id = f"patterns-{PAGE}-{VIZ_ID}", 
+                                            options = [
+                                                {"value": "bot", "label": "bot"},
+                                            ], 
+                                            value = ['bot'],
```

Comment:
@JamesKunstle thoughts on this being default to filter out the string bot? Maybe something like "potential-bot-filter" ? or completely blank?

Reply:
@christinaexyou @JamesKunstle